src/4.18.0-513/Makefile

```diff
@@ -8,9 +8,9 @@ fuse-y := dev.o dir.o file.o inode.o control.o xattr.o acl.o readdir.o
 fuse-$(CONFIG_FUSE_DAX) += dax.o
 
 all:
-	$(MAKE) -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules
+	$(MAKE) -C $(KERNELDIR) M=$(PWD) modules
```

Comment:
I think we need to slightly improve it and use the current path, if KERNELDIR is not set
,